### DISCUSSION TITLE
Is building an integration services project on the roadmap?

### DISCUSSION THREAD

**Discussion by j-dc** [microsoft/DacFx#229] [General] [answered]
Is building an Integration Services (SSIS) project on the roadmap?
Or will this be a completely different tool?

**Comment by dzsquared** [comment, marked as answer]
hi @j-dc - SSIS projects aren't on the roadmap.  You might try reaching the core SSIS group via https://marketplace.visualstudio.com/items?itemName=SSIS.MicrosoftDataToolsIntegrationServices&ssr=false#qna or https://techcommunity.microsoft.com/t5/sql-server-integration-services/public-preview-of-ssis-projects-for-vs-2022/ba-p/3683146
